modules/con-vuln-sources.adoc

```diff
@@ -32,12 +32,15 @@ This product uses the NVD API but is not endorsed or certified by the NVD.
 
 Scanner V4 uses the following vulnerability sources:
 
-link:https://security.access.redhat.com/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: Used with release 4.6 and later. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. {product-title-short} takes advantage of VEX benefits to significantly decrease the time needed for the initial loading of vulnerability data, and the space needed to store vulnerability data.
+link:https://security.access.redhat.com/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: This source is used with release 4.6 and later. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. {product-title-short} takes advantage of VEX benefits to significantly decrease the time needed for the initial loading of vulnerability data, and the space needed to store vulnerability data.
 +
 {product-title-short} might list a different number of vulnerabilities when you are scanning with a {product-title-short} version that uses OVAL, such as {product-title-short} version 4.5, and a version that uses VEX, such as version 4.6. For example, {product-title-short} no longer displays vulnerabilities with a status of "under investigation," while these vulnerabilities were included with previous versions that used OVAL data.
 +
+For containers from the Red Hat Ecosystem Catalog, {product-title-short} provides an option to show only vulnerabilities from VEX data. VEX data for Red Hat images is the most accurate because vulnerabilities in those images have been vetted by the Red Hat security team and the results reported in VEX. Other vulnerability sources such as OSV can report vulnerabilities that Red Hat has determined are not applicable to the image. This can cause false positives in vulnerability results. Red Hat also uses VEX to provide link:https://access.redhat.com/security/middleware_security_scanning_problem[accurate security data for Middleware products].
```

Comment:
```suggestion
For containers from or built on top of container from the Red Hat Ecosystem Catalog, {product-title-short} provides an option to show only vulnerabilities from Red Hat's VEX data. VEX data for Red Hat images is the most accurate because vulnerabilities in those images have been vetted by the Red Hat security team. Other vulnerability sources such as OSV can report vulnerabilities that Red Hat has determined are not applicable to the image. This can cause false positives in vulnerability results. Enabling the setting to only use VEX data for Red Hat images will minimize these false positives.
```